tests/cypress/e2e/actions_tasks/case_68_saving_settings_local_storage.js

```diff
@@ -54,16 +76,33 @@ context('Saving setting to local storage.', () => {
 
         cy.closeSettings();
     }
+    function testGammaSetting() {
```

Comment:
I dont think we need to test internal structure of clientSettings local storage. It can change in time, but the feature can still be working. Its better to check that gamma filter is still applied after reload

Reply:
By checking slider value